app/src/ui/history/commit-list-item.tsx

```diff
@@ -391,19 +391,19 @@ export class CommitListItem extends React.PureComponent<
           ? `Squash ${count} Commits…`
           : `Squash ${count} commits…`,
         action: this.onSquash,
+        enabled: !this.props.disableSquashing,
```

Comment:
Noticed when testing the availability of squashing/reordering/cherry-picking that squashing via the context menu was not being disabled - only through drag and drop was being disabled. 🤦  This is a fix. :)

Reply:
> Noticed when testing the availability of squashing/reordering/cherry-picking that squashing via the context menu was not being disabled - only through drag and drop was being disabled. 🤦 This is a fix. :)

`[
Duplicate of #](url)`